UKHO.PKSWireMockService.API/__files/request1.json

```diff
@@ -1,6 +1,6 @@
 [
     {
         "productName": "GB301910",
-        "edition": "9"
+        "edition": "10"
```

Comment:
Is this value dependent on the data or does it change often? If so should this be library variable or somewhere we can better manage it?

Reply:
This data is prepared as per ESS and FSS mock service , as the data in mock we updated as per the changes required with each functionality.

Now once the previous all dependent PBIs are through, won't require the change later.